File: Codes/Dalily_Flash/Week_3/1Feb/Java/Program5.java
import java.util.*;

class GCD {

	public static void main(String[] args) {

		Scanner sc = new Scanner(System.in);
		System.out.print("Enter No1 = ");
		int no1 = sc.nextInt();
		System.out.print("Enter No2 = ");
		int no2 = sc.nextInt();
		int temp=0;
		for(int i=1;i<=no1 && i<=no2;i++) {

			if(no1%i==0 && no2%i==0) {

				temp=i;
			}
		}

		System.out.println("The GCD of "+no1+" & "+no2+" is "+temp);
	}
}

File: Codes/Dalily_Flash/Week_3/1Feb/Python/Program5.py
no1 = int(input("Enter No1 = "))
no2 = int(input("Enter No2 = "))
if(no1>no2):
    no=no2
else:
    no=no1
for n in range(1,no+1):
    if(no1%n==0) and (no2%n==0):
        gcd = n  
print("The GCD of",no1,"and",no2,"is",gcd)
